crates/rbuilder/src/building/built_block_trace.rs

```diff
@@ -164,4 +164,14 @@ impl BuiltBlockTrace {
 
         Ok(())
     }
+
+    pub fn transaction_hashes(&self) -> Vec<TxHash> {
+        let mut hashes = Vec::new();
```

Comment:
nit: you can preallocate at least self.included_orders + some small number

Reply:
see nit below where I propose to use u64 computed as ahash from &B256 txs

Reply:
or you can XOR last limb of tx hashes into u64 since tx hashes are already hashes